### DISCUSSION TITLE
TAG v0.3.0 and lat BRANCH release-0.3 commit are not the same

### DISCUSSION THREAD

**Discussion by luarx** [prometheus-operator/kube-prometheus#602] [General]
**Last commit of release-0.3 branch** and **TAG v0.3.0** are not the same.

Which commit should I use to use the v0.3 version? (as the new ones are not compatible with my K8s cluster version)

Thanks a lot!

**Comment by metalmatze** [comment]
This branch doesn't need to be the same as the tagged release.
We can move ahead on the release branch and add new things and eventually decide to tag a commit on that release branch with v0.3.1 for example. 

Which one you use is up to you. We mostly use the `release-0.3` branch, as updating via `jb update` is easier.
